### PR TITLE
Update setup instructions following feedback

### LINKED ISSUE #13: Update setup instructions

### PR DESCRIPTION
**Pull request by Francis-Gurr**

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -10,14 +10,16 @@ If you get stuck do not worry, you can try ask [ChatGPT](https://chatgpt.com/) f
 ## 1. Download and install Git
 [Git](https://git-scm.com/) is a version control system that tracks changes in your code, allowing multiple people to collaborate and manage project versions efficiently.
 
+First check whether you already have git installed by running `git --version` in your terminal. It should print a version number if you already have git installed.
```
Comment (Gemma418) [inline docs/setup.md]: Should this be in the mac specific section?

Comment (Francis-Gurr) [reply docs/setup.md]: No this is the same command in mac and windows to check whether git is installed

Comment (Francis-Gurr) [reply docs/setup.md]: I have updated the wording of the line to hopefully make it clearer that the Mac/Windows instructions are just if you haven't installed and to skip to the second part if you already have it installed

**Review by Madeleinvm — APPROVED** [review]
LGTM

**Pull request merged**